examples/data-transfer-mssql/main.py

```diff
@@ -67,7 +69,7 @@ def get_connection(db_host, db_name, db_username, db_password):
     return pyodbc.connect(connection_str, timeout=300)
 
 
-def populate_data(RECORD_COUNT: int, db_cursor):
+def populate_data(RECORD_COUNT: int, db_cursor: Cursor) -> None:
```

Comment:
See this - what I meant to say was to omit return type if they don't have a return type i.e if they return nothing.

![image](https://user-images.githubusercontent.com/3315763/69882196-8b35c380-12f7-11ea-9e9b-b1b0a59e4596.png)

```suggestion
def populate_data(RECORD_COUNT: int, db_cursor: Cursor):
```

This would be a `void`  function it returns `None`.